src/js/jsx/help/FirstLaunch.jsx

```diff
@@ -62,6 +63,18 @@ define(function (require, exports, module) {
             }
         },
 
+        /**
+         * Calls for the URL to be opened and then logs the headlights data
+         *
+         * @param {string} url
+         * @param {string} name
+         *
```

Comment:
`@private`. Also should This instead call the flux action `menu.openURL`? Then you get logging for free!